tests/integration/llm/client.py

```diff
@@ -903,8 +903,10 @@ def test_handler_rolling_batch(model, model_spec):
     if "adapters" in spec:
         req["adapters"] = spec.get("adapters")[0]
     logging.info(f"req {req}")
-    res = send_json(req)
-    logging.info(f"res: {res.content}")
+    res = send_json(req).content.decode("utf-8")
```

Comment:
Why we need this? Are we changing req in send_json?

Reply:
no, just decode to utf8 for pattern matching

Reply:
It's better separate in two line.